src/hub.js

```diff
@@ -1556,6 +1556,7 @@ document.addEventListener("DOMContentLoaded", async () => {
   hubPhxChannel.on("hub_refresh", ({ session_id, hubs, stale_fields }) => {
     const hub = hubs[0];
     const userInfo = hubChannel.presence.state[session_id];
+    const displayName = (userInfo && userInfo.metas[0].profile.displayName) || "API";
```

Comment:
Add comment about this being the first (of possibly many) places where we do something differently based on whether the message came from the API vs from another user on the channel.